docker/localnode/config/price_feeder_config.toml

```diff
@@ -17,7 +17,7 @@ threshold = "2"
 base = "ATOM"
 chain_denom = "uatom"
 providers = [
-  "binance",
+  "huobi",
```

Comment:
for my info, why huobi locally?

Reply:
Not really sure, just update to reflect what we use in devenet-3. @udpatil might know this? I believe binance is actually not connectable at this point, connections are being rejected

Reply:
I believe the binance websocket has had issues more recently, so this is probably just to ensure liveness of data sources by using one that is more reliably available